README.md

```diff
@@ -75,6 +75,31 @@ model = nn.Sequential(
 )
 ```
 
+## Frequently Asked Questions
+Before posting an issue, please go through the following troubleshooting steps:
+- Check whether the issue is torchsparse specific or enviornment specific.
```

Comment:
A small typo: "enviornment" => "environment" (there are a few more occurrences in the following paragraphs).

Reply:
Besides, let's refer to this package as "TorchSparse".

Reply:
Thanks! I always misspell that.